.github/workflows/ci.yml

```diff
@@ -5,7 +5,7 @@ name: CI
 
 on:
   push:
-    branches: [ develop ]
+    branches: [ develop, feature/** ]
```

Comment:
Maven cannot build the project when there is a push on a feature branch because the env variables are missing during the build. Don't know why the env variables are missing here but not during a build triggered by a pull request

Reply:
We talked about this last on Friday ... I don't know why, but after triggering the CI build manually again. The env variables are available to the workflow. This is a strange new behaviour from GitHub. So if you see something like this again, just try to manually trigger the build again.

Reply:
Strange behavior because I restarted the run and it still failed. Maybe it's because you're an admin ...